source/_code.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <button aria-controls="example-snippets"
```

Comment:
TODO before merging: These `example` ids should be pulled through from the `code_for` argument.